projects/js-packages/charts/package.json

```diff
@@ -111,7 +111,7 @@
 		"sass-loader": "^13.0.0",
 		"storybook": "8.6.7",
 		"tsconfig-paths-webpack-plugin": "4.2.0",
-		"typescript": "5.7.2",
+		"typescript": "5.8.2",
```

Comment:
It seems like we had two versions of typescript in the repo.

Reply:
It happens. When I see PRs where someone is adding new versions of dependencies I add a code review suggesting a match with the existing monorepo version, but I don't see all PRs.

Reply:
> I don't see all PRs

😱